client/src/components/cart/shippItem/index.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { FC } from 'react';
+import { cancleSVG, modifySVG, underlineShortSVG } from '~/assets';
+import Checkbox from '~/components/common/Checkbox';
+import { confirm } from '~/utils/modal';
+import { ShipType } from '~/components/cart/shipping';
+
+import {
+  ShipItemWrapper,
+  CheckboxWrapper,
+  ShipUser,
+  ShipAddress,
+  ShipControlWrapper,
+  ShipControl,
+  ShipHover,
+} from './index.style';
+
+interface Props {
+  shipItem: ShipType;
+  selected: boolean;
+  changeSelectedBtn: (shipIdx: number) => void;
+  removeShippingItem: (shipIdx: number) => void;
+  modifyBtnClick: (shipIdx: number) => void;
+}
+
+const ShipItem: FC<Props> = ({
+  shipItem,
+  selected,
+  changeSelectedBtn,
+  removeShippingItem,
+  modifyBtnClick,
+}) => {
+  const handleSelectBtnClick = () => {
+    changeSelectedBtn(shipItem.idx);
+  };
+
+  const handleRemoveBtnClick = () => {
+    confirm('정말 삭제하시겠어요?', () => {
+      removeShippingItem(shipItem.idx);
+    });
+  };
+
+  const handleModifyBtnClick = () => {
+    modifyBtnClick(shipItem.idx);
+  };
+
+  return (
+    <ShipItemWrapper>
+      <CheckboxWrapper onClick={handleSelectBtnClick}>
+        <Checkbox checked={selected} />
+      </CheckboxWrapper>
+      <ShipUser>
+        <div>{shipItem.name}</div>
+        <div>{shipItem.phone}</div>
+      </ShipUser>
+      <ShipAddress>{shipItem.address}</ShipAddress>
+      <ShipControlWrapper>
+        <ShipHover>
+          <ShipControl onClick={handleModifyBtnClick}>
+            <img src={modifySVG} alt="modify" />
+            <div>수정</div>
```

Comment:
저는 최대한 블록 레벨인 요소에만 div를 넣으려고 합니다
수정 Text는 ShipControll로 감싸지는 inline 요소라 느껴져서 저는 span을 사용했을 것 같아요...!
이것도 다른 분들 의견이 궁금하네요

```suggestion
            <span>수정</span>
```

Reply:
음,,,, style에 따라 다르지 않을까요. 수정이라는 요소의 display를 inline이 아니라 block으로 의도한 것이라면 div가 맞고 inline이라면 span이 맞다고 생각합니다.